app/scripts/lib/ppom/ppom-middleware.ts

```diff
@@ -0,0 +1,43 @@
+import { PPOM } from '@blockaid/ppom';
+
+import { PPOMController } from '@metamask/ppom-validator';
+
+const ConfirmationMethods = [
```

Comment:
Should we freeze this array? `Object.freeze([...])` (just to be sure that no dep is changing the whitelisted methods)